interpreters/luajit/Makefile

```diff
@@ -0,0 +1,170 @@
+############################################################################
+# apps/interpreters/luajit/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(APPDIR)/Make.defs
+
+LUAJIT_PATCHS ?= $(sort $(wildcard 000*.patch))
+
+LUAJIT_VERSION := $(patsubst "%",%,$(CONFIG_LUAJIT_VERSION))
+LUAJIT_UNPACK   = LuaJIT
+LUAJIT_TARBALL  = $(LUAJIT_VERSION).tar.gz
+LUAJIT_URL_BASE = https://github.com/LuaJIT/LuaJIT/archive
+LUAJIT_URL      = $(LUAJIT_URL_BASE)/$(LUAJIT_TARBALL)
+
+MAINSRC = luajit.c
+
+CSRCS += lib_aux.c lib_base.c lib_bit.c lib_buffer.c lib_debug.c
```

Comment:
can we sort file name to help verify with the package

Reply:
> can we sort file name to help verify with the package

What do you mean by that?

Reply:
single or multiple file per line is fine, but the file name is alphabeticaly arranged.